### DISCUSSION TITLE
Feature idea: Hotkeys for sending a window to another desktop

### DISCUSSION THREAD

**Discussion by henriklewander** [valinet/ExplorerPatcher#650] [Ideas]
Hello and thanks for this amazing project!

Today it is possible to switch to next desktop with Ctrl + Windows key + Arrow keys.
It would be great to be able to send a window to next desktop, I would suggest using Alt + Windows key + Arrow keys.

Is this perhaps outside of scope for this project? In any case it would be super helpful.

**Comment by Andre-10** [comment]
It is already built in Windows. Press **Shift+Winkey+Arrow** to send an App Window to another desktop. Unfortunately, doesn't work for Virtual desktops, only multi monitor

**Reply by valinet** [reply]
So it doesn’t work as the OP suggested, so there isn’t something built in after all, right?

**Reply by Andre-10** [reply]
It works for physical desktops, not for virtual. Not sure what was the OP request. (my mistake)
At first, I wasn't even thought about virtual desktops.. came into mind while I was typing my answer. 

Would be great if you can extend this functionality to Virtual desktops, thou 

Have to correct myself.. Ctrl+Winkey+Arrow works for Virtual desktops, so my assumption was wrong. You can delete this if you want, I usually don't...

**Reply by henriklewander** [reply]
Yeah, virtual desktop was my goal, sorry for not spelling it out. Would be super helpful to have keyboard shortcut for sending windows to next virtual desktop.

**Reply by henriklewander** [reply]
@valinet Is this something that can be considered to be added? 
Ctrl-Winkey-Arrow works for switching virtual desktops but there is no shortcut key for sending the current window to another virtual desktop.

**Comment by Poopooracoocoo** [comment]
https://github.com/hwtnb/SylphyHornPlusWin11 has this I think.
